src/main/createStore.ts

```diff
@@ -63,6 +67,29 @@ export function createStore(initialState: SiteDataBySiteID = {}): Store {
 				}
 			};
 		},
+
+		/**
+		 * Merges in new image data for a single image associated with a site
+		 *
+		 * @param siteID
+		 * @param imageID
+		 * @param newState
+		 */
+		setStateByImageID(siteID, imageID, newState) {
```

Comment:
Minor but I wonder about the consistency of our naming conventions -- `siteId` Vs `siteID` and `imageID` vs `imageId`

Reply:
Same over here actually. I think I prefer `siteId`, but for some reason, went with `siteID` in the early days of the repo and never changed it 😬